src/main/java/com/ericsson/eiffel/remrem/shared/CommonMsgService.java

```diff
@@ -0,0 +1,13 @@
+package com.ericsson.eiffel.remrem.shared;
+
+public class CommonMsgService {
```

Comment:
I would prefer to have separate classes for different purposes. Hence calling a class CommonMsgService doesn't tell anything about what this class is used for. I guess we would name it with something about Jar File Loading".